### PR TITLE
Feature/deseng55

### PR DESCRIPTION
**Pull request by Baelx**
Should be the last PR in this project for this ticket.

[-] Wrote a test or tests for code you added.
[x] Linted the files you touched.
[x] Wrote a doc block for each method you added.
[-] Add documentation to the README

### CODE REVIEW

File: src/app/models/project.ts [hunk 1]
```diff
@@ -96,37 +98,22 @@ export class Project {
...
+  }
+}
 
-    // // copy documents
-    // if (obj && obj.documents) {
-    //   for (const doc of obj.documents) {
-    //     this.documents.push(doc);
-    //   }
-    // }
+export interface ProjectLogo {
```
Comment (ASpiteri-BCGov) [inline src/app/models/project.ts:110]: Maybe consider creating a types file where all types are defined and exported.

**Review by ASpiteri-BCGov — APPROVED** [review]

File: src/app/models/project.ts (hunk 1 above)
Comment (Baelx) [reply src/app/models/project.ts:110]: Thanks for the suggestion. I definitely agree we should be keeping these types as organized as possible.

This project.ts file - and all files within the models directory - contains the types for all of the "data structures" in the app. Just based on how we've organized types in this app, I think any "project" type should be fine to go within project.ts

File: src/app/project/project.component.scss [hunk 2]
```diff
@@ -30,6 +30,23 @@
...
+        padding: 0;
+        list-style-type: none;
+        display: flex;
+
+        li {
+          margin-right: 1rem;
+
+          a {
+            img {
+              max-height: 90px;
```
Comment (SteveMHoward) [inline src/app/project/project.component.scss:44]: Have you verified this works as expected in Chrome, Firefox and Safari? 

I am not sure if it has changed, but previously there was an issue using just max-height without defining a height in one of the major non-IE browsers (I think it was FF but don't recall) - the height would get returned as 0px.

Comment (Baelx) [reply src/app/project/project.component.scss:44]: Interesting! I haven't run across that bug before. Just for good measure, I had a look at this in those browsers and everything it tickety boo

**Review by SteveMHoward — APPROVED** [review]

**Pull request merged**